core-bundle/src/Resources/contao/languages/en/tl_files.xlf

```diff
@@ -129,7 +129,7 @@
         <source>The synchronization has been completed</source>
       </trans-unit>
       <trans-unit id="tl_files.syncResult">
-        <source>&lt;strong&gt;Result:&lt;/strong&gt; %s added | %s modified | %s unchanged | %s moved | %s deleted</source>
+        <source>&lt;strong&gt;Result:&lt;/strong&gt; %s added | %s updated | %5$s deleted</source>
```

Comment:
`%5$s`?

Reply:
That means use the 5th parameter instead of the 3rd. This is to make the label compatible with older translations. (because the 5th parameter was used for “deleted” files previously).

Reply:
So does this mean that we are no longer tracking modified, unchanged and moved files?

Reply:
Modified and moved is consolidated into updated AFAIK. The count of unchanged files is probably not that valuable as it is just the count of all files minus updated and added.